### ISSUE TITLE
Let me Make Gem sets that get automatically socketed into gear.

### ISSUE THREAD

**Issue by Jodsderechte** [Larsj02/Remix-Gem-Helper#9]
Hello! I have no idea how feasable this is since i haven't looked into the gear api. 
However getting gear upgrades is quite tedious while leveling since you have to socket in the same ges many times. It would be nice if i could save a set of gems and when i equip new items with empty slots it would automatically socket in the gems from my inventory into it. In a perfect world it would even outsocket them from my old gear but idk what part of those functions is hardware protected maybe it doesn't even work

**Comment by Larsj02** [comment]
This was already one of my first thoughts when making the addon sadly that is not possible as each unsocketing is protected and can only be executed by clicking a secure action button

**Comment by Jodsderechte** [comment]
technically showing a button in the middle on the screen that you have to click to make the switches similarly to what [new openables](https://legacy.curseforge.com/wow/addons/new-openables) worked would be an alternative. I personally would use that but it's your decision ofc

**Comment by Jodsderechte** [comment]
I see you added this in fd4afb7 there seems to be a bug however in today's release when you scrap multiple items with gems at the same time. It just overwrites the old buttons instead of adding them together

**Comment by Larsj02** [comment]
Yes like i said before this was an early release just to get some features out in a future release I'll clean stuff up and probably add something that keeps track of which slot an item was placed in and then if it got destroyed prompt for the resocket
